requirements-igwn.txt

```diff
@@ -0,0 +1,6 @@
+# For LDG service access
+ciecplib>=0.4.4
+dqsegdb2>=1.0.1
+amqplib
+htchirp >= 2.0
```

Comment:
I thought this one had been added by Duncan? Not sure what this is doing at all?